.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 .hubot_history
 npm-debug.log
 secrets.yml
+deploy.yml
```

Comment:
Instead of the local directory, please use the `/tmp` directory. Otherwise, Semaphore will cache the generated `deploy.yml` file and potentially cause some maintainer nightmare.

Reply:
off to fix that 👍